openedx/features/custom_fields/multiselect_with_other/db/fields.py

```diff
@@ -73,15 +74,15 @@ def formfield(self, **kwargs):
         return MultiSelectWithOtherFormField(**defaults)
 
     def validate(self, value, model_instance):
-        """
-        This function is to validate the input values for multi select field,
+        """ This function is to validate the input values for multi select field,
```

Comment:
@zahid-arbisoft Is it fine to start the docstring on same line for multi line docstring ?
As in your PR, I see, you started multi line docstring with a line break .. like it was before 
```
       """
        This function is to validate the input values for multi select field,
```
instead of 
```
""" This function is to validate the input values for multi select field,
```

Reply:
Yes, I have asked @abdullah998 to update all docstrings, I did not know before, this is what Edx recommends.